dry-run/symphonik/SymphonIK.py

```diff
@@ -0,0 +1,227 @@
+#             "number_assets": 25,
+#             "sort_key": "quoteVolume",
+#             "refresh_period": 1800
+#         },
+#         {"method": "AgeFilter", "min_days_listed": 10},
+#         {"method": "PrecisionFilter"},
+#         {"method": "PriceFilter", "low_price_ratio": 0.001},
+#         {
+#             "method": "RangeStabilityFilter",
+#             "lookback_days": 3,
+#             "min_rate_of_change": 0.1,
+#             "refresh_period": 1440
+#         },
+#     ],
+
+def ssl_atr(dataframe, length=7):
+    df = dataframe.copy()
+    df['smaHigh'] = df['high'].rolling(length).mean() + df['atr']
+    df['smaLow'] = df['low'].rolling(length).mean() - df['atr']
+    df['hlv'] = np.where(df['close'] > df['smaHigh'], 1,
+                         np.where(df['close'] < df['smaLow'], -1, np.NAN))
+    df['hlv'] = df['hlv'].ffill()
+    df['sslDown'] = np.where(df['hlv'] < 0, df['smaHigh'], df['smaLow'])
+    df['sslUp'] = np.where(df['hlv'] < 0, df['smaLow'], df['smaHigh'])
+    return df['sslDown'], df['sslUp']
+
+
+def create_ichimoku(dataframe, conversion_line_period, displacement, base_line_periods, laggin_span):
+    ichimoku = ftt.ichimoku(dataframe,
+                            conversion_line_period=conversion_line_period,
+                            base_line_periods=base_line_periods,
+                            laggin_span=laggin_span,
+                            displacement=displacement
+                            )
+    dataframe[f'tenkan_sen_{conversion_line_period}'] = ichimoku['tenkan_sen']
+    dataframe[f'kijun_sen_{conversion_line_period}'] = ichimoku['kijun_sen']
+    dataframe[f'senkou_a_{conversion_line_period}'] = ichimoku['senkou_span_a']
+    dataframe[f'senkou_b_{conversion_line_period}'] = ichimoku['senkou_span_b']
+
+
+class SymphonIK(IStrategy):
+    # Optimal timeframe for the strategy
+    timeframe = '1m'
+
+    # generate signals from the 5m timeframe
+    informative_timeframe = '1m'
+
+    # WARNING: ichimoku is a long indicator, if you remove or use a
+    # shorter startup_candle_count your results will be unstable/invalid
+    # for up to a week from the start of your backtest or dry/live run
+    # (180 candles = 7.5 days)
+    startup_candle_count = 444  # MAXIMUM ICHIMOKU
+
+    # NOTE: this strat only uses candle information, so processing between
+    # new candles is a waste of resources as nothing will change
+    process_only_new_candles = True
+
+    minimal_roi = {
+        "0": 10,
+    }
+
+    # WARNING setting a stoploss for this strategy doesn't make much sense, as it will buy
+    # back into the trend at the next available opportunity, unless the trend has ended,
+    # in which case it would sell anyway.
+
+    # Stoploss:
+    stoploss = -0.10
+
+    def informative_pairs(self):
+        pairs = self.dp.current_whitelist()
+        informative_pairs = [(pair, self.informative_timeframe)
+                             for pair in pairs]
```

Comment:
Delete this, as informative_timeframe = timeframe

Reply:
Do you mean that this line should be swapped by what you said? Isn't informative_pairs needed?

Reply:
We can remove this line and the informative_pairs variable I think

Reply:
I believe we can remove the whole function because it's not being called anywhere